server.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+const errorHandler = (err, req, res, next) => {
+  console.log(err);
+  res.status(500).send(err.message);
+};
+
+app.route("/ping").get((req, res, next) => {
```

Comment:
It's a good practice to separate your routes into a different file or module. This will make your server.js file cleaner and more maintainable. [important]